.sync/Version.njk

```diff
@@ -33,8 +33,8 @@
 {% set mu_devops = "v13.0.3" %}

```

Comment:
Hold this PR until mu_feature repos have switch CI to 202502.

Reply:
Going to approve now so you can merge when ready.